website/markdown/docs/contribution/manifesto.mdx

```diff
@@ -45,14 +45,13 @@ Xcode is a great tool, but so many years of improvements, new platforms, and pro
 
 Tuist should take the opportunity to keep things simple because working on simple things is fun and motivates us. No one wants to spend time trying to debug an error that happens at the very end of the compilation process, or understanding why they are not able to run the app on their devices. Xcode delegates the tasks to its underlying build system and in some cases it does a very poor job translating errors into actionable items. Have you ever got a “framework X not found” error and you didn’t know what to do? Imagine if we got a list of potential root causes for the bug.
 
-## 5. We are not a project generation tool
+## 5. Start from the developers' experience
 
-The generation of Xcode projects is a mean and not an end in itself. Our goal is not to provide developers with another language to define their projects but rather, help them cope with the challenges they face when they scale up their projects. That means we might not provide APIs for all the features supported by Xcode, and that's fine. If we did, we'd be re-writing Xcode projects in a new language, and that's a task that falls into Apple's domain.
+Part of the reason why there is a lack of innovation around Xcode, or put differently, not as much as in other programming environments, is because **we often start analyzing problems from existing solutions.** As a consequence, most of the solutions that we find nowadays revolve around the same ideas and workflows. While it’s good to include existing solutions in the equations, we should not let them constrain our creativity.
 
-That means we sometimes have to say no to bringing ideas from Xcode that complicate scaling. For instance, if dealing with code signing at scale is something that Xcode doesn't do well, let's take the opportunity to re-think how code signing is done.
+We like to think as [Tom Preston](https://tom.preston-werner.com/) puts it in [this podcast](https://tom.preston-werner.com/): _"Most things can be achieved, whatever you have in your head you can probably pull of with code as long as is possible within the constrains of the universe"_. If we imagine **how we'd like the developer experience to be**, it's just a matter of time to pull it off — starting analyzing the problems from the developer experience gives us a unique point of view that will lead us to solutions that users will love to use.
```

Comment:
`you can probably pull of with code` -> `you can probably pull off with code`

`starting analyzing the` -> `by starting to analyze the`

```suggestion
We like to think as [Tom Preston](https://tom.preston-werner.com/) puts it in [this podcast](https://tom.preston-werner.com/): _"Most things can be achieved, whatever you have in your head you can probably pull off with code as long as is possible within the constrains of the universe"_. If we imagine **how we'd like the developer experience to be**, it's just a matter of time to pull it off — by starting to analyze the problems from the developer experience gives us a unique point of view that will lead us to solutions that users will love to use.
```